pkg/cmd/server/api/serialization_test.go

```diff
@@ -203,6 +203,12 @@ func fuzzInternalObject(t *testing.T, forVersion unversioned.GroupVersion, item
 				obj.NodeSelectorLabelBlacklist = []string{"kubernetes.io/hostname"}
 			}
 		},
+		func(obj *configapi.GrantConfig, c fuzz.Continue) {
+			c.FuzzNoCustom(obj)
+			if len(obj.ServiceAccountMethod) == 0 {
+				obj.ServiceAccountMethod = "prompt"
```

Comment:
This follows the defaulting rule.  Turns out that the fuzzer set values for each of the fuzzing runs while it was in the merge queue, so it round tripped fine.